src/client/mod.rs

```diff
@@ -1,19 +1,29 @@
 //! Process HTTP connections on the client.
 
 use async_std::io::{self, Read, Write};
+use http_types::headers::USER_AGENT;
 use http_types::{Request, Response};
+use lazy_static::lazy_static;
 
 mod decode;
 mod encode;
 
 pub use decode::decode;
 pub use encode::Encoder;
 
+lazy_static! {
+    static ref DEFAULT_USER_AGENT: String = format!("http-rs-h1/{}", env!("CARGO_PKG_VERSION"));
```

Comment:
Unsure what to call it, `async-h1` seems wrong when used by e.g. surf, but also maybe surf should override it anyways. Unsure.

Reply:
+1 for surf overriding or appending. why not `async-h1`, though, since that's the name of the crate? Then it could be something like `surf/2.0.0 (async-h1/6.0.0)`